packages/tina-graphql-gateway-cli/src/cmds/compile/index.ts

```diff
@@ -203,6 +209,23 @@ let types = [
 let compiledTemplates = [];
 
 export const compile = async () => {
+  // FIXME: This assume it is a schema.ts file
```

Comment:
Given we can't know how people are gonna name their collections, the default should always be a global `schema.ts` right?

Reply:
We may support `schema.js` (and we probably should) so I think this would just mean we need to support the scenario where there's neither

EDIT: I don't think we need to support the `schema.js` scenario in the scope of this PR FYI

Reply:
If the user doesn't have a tsconfig, will this still work?

Reply:
It doesn't use the `tsconfig` from the project so it should, though we should probably just rely on the user's setup rather than bringing in our own direct Typescript dependency

Reply:
It doesn't use the tsconfig from the project so it should, though we should probably just rely on the user's setup rather than bringing in our own direct Typescript dependency